sssd_test_framework/utils/pam.py

```diff
@@ -8,48 +8,12 @@
 from pytest_mh.utils.fs import LinuxFileSystem
 
 __all__ = [
-    "PAMUtils",
-    "PAMAccess",
-    "PAMFaillock",
+    "PAMAccessUtils",
```

Comment:
Why rename the classes?

Reply:
These classes were written incorrectly and setup and teardown of PAMAccess and PAMFaillock would not be called. So I fix that, then renamed the classes to follow our naming convention.